app/Providers/PlayerCountProvider.php

```diff
@@ -29,7 +29,7 @@ public function boot()
             'layouts.playercount',
             static function ($view) {
                 $playerCount = Cache::remember('playerCount', 60 * 5, static function () {
-                    return OnlineOfflineLog::where('status', OnlineOfflineLog::STATUS_LOGGED_IN)->count();
```

Comment:
@JellyBitz there is no difference between
`where('status', OnlineOfflineLog::STATUS_LOGGED_IN)->count()`
the `=` is there automatic when nothing is set.
[Laravel Docs](https://laravel.com/docs/7.x/queries#where-clauses)